### ISSUE TITLE
Allow logarithmic averaging of pressure variables when regridding, binning

### ISSUE THREAD

**Issue by StevenCompernolle** [stcorp/harp#251]
For pressure variables like cloud_pressure, application of bin_spatial (or other gridding or binning routines) is preferably done with averaging for log10(cloud_pressure) instead of cloud_pressure.
One solution is to convert to logarithmic variables (e.g., cloud_pressure_log10), apply then the bin_spatial, and convert back. Currently such operations are not allowed in HARP.
Other option could be to have an option in the binning or regridding operations that allow averaging in log space.

**Comment by svniemeijer** [comment]
Good point. We already use logarithmic-aware treatment of pressure when performing vertical regridding.
But we should indeed also apply this for spatial averaging.

**Comment by StevenCompernolle** [comment]
Any chance to include this for the next release 1.14, or is this a longer-term change? I could use it for my TROPOMI validation work.

**Comment by svniemeijer** [comment]
I will have a look. It is a busy period, but the change shouldn't be too big.
